### PR TITLE
Feature/create_retention_policy

### PR DESCRIPTION
**Pull request by dongshu2013**

### CODE REVIEW

**Comment by dongshu2013** [comment]
Changes include: 
* Bug fixes in create_retention_policy task. Sorry for that, didn't test before.
* Support specifying config file instead of using the default one. I met the problem that the config file cannot be put in the config folder.
* Fix crash problem in create_user task when not specifying user in config. Sometimes we just want to use the default username and password.

All these code changes are tested already.

**Comment by dongshu2013** [comment]
@dplummer

File: README.md [hunk 1]
```diff
@@ -26,7 +26,9 @@ your influxdb. See the example `influxdb_queries.yml` for the archive queries.
 ## Usage
 
 This library expects your influxdb config to be located in the
-`config/influxdb.yml` file. For example (change *myapp* to your application
+`config/influxdb.yml` file. You can also specify a config file
+by using the CONFIG environment variable. For example (change
```
Comment (dplummer) [inline README.md]: This should be `INFLUXDB_CONFIG_FILE`

File: README.md [hunk 2]
```diff
@@ -130,6 +132,9 @@ in the `db/influxdb_queries.yml` file.
 `rake influxdb:setup`
 Runs all the above rake tasks.
 
+`CONFIG=/your/config/file/path rake influxdb:setup`
```
Comment (dplummer) [inline README.md]: `INFLUXDB_CONFIG_FILE`

File: lib/influxdb_setup/create_user.rb [hunk 3]
```diff
@@ -8,10 +8,12 @@ def call
       root = @config.build_client
       users = root.list_users.map{|user_hash| user_hash["username"]}
 
-      unless users.include?(user)
-        root.create_database_user(db, user, pass)
-      else
+      if user == nil
```
Comment (dplummer) [inline lib/influxdb_setup/create_user.rb]: Use `.nil?` for nil checks, ie `user.nil?`

**Comment by dongshu2013** [comment]
Thanks for the merge!

**Comment by dplummer** [comment]
Thanks for the good work and code!

**Comment by dplummer** [comment]
Released v1.1.0.

**Pull request merged**